flow/src/main/java/com/sequenceiq/flow/service/flowlog/FlowChainLogService.java

```diff
@@ -105,7 +106,13 @@ public boolean hasEventInFlowChainQueue(List<FlowChainLog> flowChains) {
                     .get();
             LOGGER.debug("Checking if chain with id {} has any event in it's queue", latestFlowChain.getFlowChainId());
             LOGGER.trace("Chain string in db: {}", latestFlowChain.getChain());
-            Queue<Selectable> chain = (Queue<Selectable>) JsonReader.jsonToJava(latestFlowChain.getChain());
+            Queue<Selectable> chain;
+            if (null != latestFlowChain.getChainJackson()) {
```

Comment:
The entity field `FlowChainLog.chainJackson` is marked as `nullable = false`, so I wonder how this can work. Perhaps checking for `Strings.notEmpty()` would be better?

Reply:
if the `chainJackson` is null that means that this chain was created with a version of CB that did not have this column